test/unit_tests/test_ci_test.py

```diff
@@ -1,85 +1,113 @@
+        all_tests_result.tests_are_ok = tests_are_ok
+        all_tests_result.command_line_output_path = create_autospec(Path)
+        return all_tests_result
+
+    def run_execute_tests(self):
+        self.ci_execute_test.execute_tests(
+            flavor_path=(self.flavor_path,),
             docker_user=None,
             docker_password=None,
-            test_container_folder=TEST_CONTAINER,
+            test_container_folder=self.test_container_folder,
         )
-    assert print_file_function_mock.mock_calls == [
-        call(db_tests_all_tests_result.command_line_output_path),
-        call(linker_namespace_all_tests_result.command_line_output_path)] \
-           and print_docker_images_function_mock.mock_calls == [call()] \
-           and run_db_test_mock.mock_calls == [
-               call(flavor_path=(FLAVOR,), test_folder=(), release_goal=('release',), workers=7,
-                    source_docker_username=None, source_docker_password=None, test_container_folder=TEST_CONTAINER),
-               call(flavor_path=(FLAVOR,), workers=7, test_folder=('test/linker_namespace_sanity',),
-                    release_goal=('base_test_build_run',), source_docker_username=None, source_docker_password=None,
-                    test_container_folder=TEST_CONTAINER)
-           ]
-
-
-def create_all_tests_result_mock(tests_are_ok: bool):
-    all_tests_result: Union[MagicMock, AllTestsResult] = create_autospec(AllTestsResult)
-    all_tests_result.tests_are_ok = tests_are_ok
-    all_tests_result.command_line_output_path = Mock()
-    return all_tests_result
+
+    @pytest.fixture()
+    def run_db_tests_calls(self):
+        return [
+            call.run(flavor_path=(self.flavor_path,), test_folder=(), release_goal=('release',), workers=7,
+                     docker_username=None, docker_password=None,
+                     test_container_folder=self.test_container_folder),
+            call.run(flavor_path=(self.flavor_path,), workers=7, test_folder=('test/linker_namespace_sanity',),
+                     release_goal=('base_test_build_run',), docker_username=None, docker_password=None,
+                     test_container_folder=self.test_container_folder)
+        ]
+
+
+class TestIntactFlavor(BaseCIExecuteTest):
+
+    @pytest.fixture
+    def setup_intact_flavor(self, setup_ci_execute_test):
+        self.db_tests_all_tests_result = self.create_all_tests_result_mock(tests_are_ok=True)
+        self.linker_namespace_tests_all_tests_result = self.create_all_tests_result_mock(tests_are_ok=True)
+        mock_cast(self.db_test_runner_mock.run).side_effect = [self.db_tests_all_tests_result,
+                                                               self.linker_namespace_tests_all_tests_result]
+
+    @pytest.fixture
+    def run(self, setup_intact_flavor):
+        return self.run_execute_tests()
+
+    def test_ci_step_output_printer_call(self, run):
```

Comment:
Is Parameter `run` unused intentionally?

Reply:
This is a fixture which should run, but I don't need anything from it, because it doesn't have a return value